src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -220,7 +220,9 @@ class CrossDeviceLinkUI extends Component {
               </Button>
             </div>
           </div>
-          { invalidNumber && <div className={style.numberError}>{translate('errors.invalid_number.message')}</div> }
+          <div role="alert" aria-atomic="true">
```

Comment:
I don't think you need `aria-atomic="true"` here because `role="alert"` is equivalent to setting aria-live="assertive" and aria-atomic="true".

Reply:
Added it in as I read in the W3C documentation that "`aria-atomic=true` is necessary to make Voiceover on iOS read the error messages after more than one invalid submission." which I felt might happen in this case. Happy to remove if you think it's unlikely or I've misinterpreted the doc explanation.

https://www.w3.org/WAI/GL/wiki/Using_ARIA_role%3Dalert_or_Live_Regions_to_Identify_Errors

Reply:
Fair enough. I didn't know that :)